src/app/app-routing.module.ts

```diff
@@ -1,17 +1,17 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-import { HomePageComponent } from './home-page/home-page.component';
+import { RecipesCardsComponent } from './recipes/recipes-cards/recipes-cards.component';
 import { RecipesEditComponent } from './recipes/recipes-edit/recipes-edit.component';
-import { RecipesComponent } from './recipes/recipes.component';
+import { RecipeResolver } from './recipes/resolvers/recipe.resolver';
 import { NotFoundPageComponent } from './shared/not-found-page/not-found-page.component';
 
 const routes: Routes = [
-	{ path: '', component: HomePageComponent },
+	{ path: '', component: RecipesCardsComponent, resolve: [RecipeResolver] },
```

Comment:
in app routing its best to have only lazy loaded modules
https://angular.io/guide/lazy-loading-ngmodules
so here you add module and inside it create separate routing for that module